src/idle.c

```diff
@@ -66,6 +66,8 @@ int idle_requested(struct fmt_main *format)
 	if (strstr(format->params.label, "-opencl"))
 		return 0;
 #endif
+	if (strstr(format->params.label, "-ztex"))
+		return 0;
```

Comment:
Could check `#ifdef HAVE_ZTEX` here for consistency with what we do for OpenCL, but that's optional. We're not doing it consistently across the codebase anyway.